app/models/daos/UserDAOImpl.scala

```diff
@@ -58,8 +59,20 @@ class UserDAOImpl extends UserDAO {
 object UserDAOImpl {
   val db = play.api.db.slick.DB
   val userTable = TableQuery[UserTable]
-//  val auditTaskTable = TableQuery[AuditTask]
-//  val labelTable = TableQuery[Label]
+  val auditTaskTable = TableQuery[AuditTaskTable]
+  val auditTaskEnvironmentTable = TableQuery[AuditTaskEnvironmentTable]
+  val auditTaskInteractionTable = TableQuery[AuditTaskInteractionTable]
+
+  val anonUsers = for {
+    (_ate, _at) <- auditTaskEnvironmentTable.innerJoin(auditTaskTable).on(_.auditTaskId === _.auditTaskId)
+    if _at.userId === "97760883-8ef0-4309-9a5e-0c086ef27573" && _at.completed === true
```

Comment:
Could you go through the code and remove all the hard coded string literals for the anon user?